_layouts/default.html

```diff
@@ -61,9 +61,6 @@
         <paper-item label="Developer API (javadoc)" icon="open-in-browser">
           <a href="api/latest/" target="_blank"></a>
         </paper-item>
-        <paper-item label="Releases &amp; Versioning">
```

Comment:
If you don't mind re-running MOE, I'd fix this internally and re-do this one. Or, you can just follow-up quickly.  I encourage you to not try to fix in place in this commit stream, as it'll mess up MOE's view of the world.